### ISSUE TITLE
`sympy-bot/release-notes` CI jobs are hanging

### ISSUE THREAD

**Issue by brocksam** [sympy/sympy#25172]
PRs are currently blocked by the `sympy-bot/release-notes`, which is hanging. Last instance of it working [appears to have been at around 14:00UTC 24/05/2023](https://github.com/sympy/sympy/pull/25168#issuecomment-1561130975).

**Comment by brocksam** [comment]
Related to https://github.com/sympy/sympy/issues/25155. On further reading it sounds like this might be an intermittent issue.

**Comment by brocksam** [comment]
Appears to be working again as of 10:00UTC 26/05/2023. For PRs where it was hanging I found removing the release notes to trigger a bot failure, and then reading them to get bot approval worked.

**Comment by brocksam** [comment]
Closing as this is no longer relevant.
